ontology/yaml/resources/HVAC/entity_types/FCU.yaml

```diff
@@ -745,6 +745,31 @@ FCU_DFMSS_DSP_ZHM_RTM_CHWZC_CHWDT:
   - CHWZC
   - CHWDT
 
+FCU_DFVSMC_CHWBZS:
```

Comment:
Please adjust this type and the following type once corrections are made to implemented abstract types.

Reply:
Related to previous abstract type points listing. This has been corrected.